workflow/controller/operator.go

```diff
@@ -1320,7 +1328,7 @@ func inferFailedReason(pod *apiv1.Pod) (wfv1.NodePhase, string) {
 			// executor is expected to annotate a message to the pod upon any errors.
 			// If we failed to see the annotated message, it is likely the pod ran with
 			// insufficient privileges. Give a hint to that effect.
-			return wfv1.NodeError, fmt.Sprintf("%s; verify serviceaccount %s:%s has necessary privileges", msg, pod.Namespace, pod.Spec.ServiceAccountName)
+			return wfv1.NodeError, fmt.Sprintf("%s%s verify serviceaccount %s:%s has necessary privileges", msg, splitChar, pod.Namespace, pod.Spec.ServiceAccountName)
```

Comment:
in the case of 403 errors, the error message as actually enough to point out the problem, so I think we should just return `msg` here